src/thunderbots/software/ai/hl/stp/action/move_action.h

```diff
@@ -33,16 +34,16 @@ class MoveAction : public Action
      * the destination
      * @param final_speed The final speed the robot should have at the destination
      * @param enable_dribbler Whether or not to enable the dribbler
-     * @param enable_autokick This will enable the "break-beam" on the robot, that will
-     *                        trigger the kicker to fire as soon as the ball is in front
-     *                        of it
+     * @param autokick This will enable the "break-beam" on the robot, that will
+     *    *                 trigger the kicker to fire as soon as the ball is in front of
+     * it
```

Comment:
formatting, also we should probably say "trigger the chipper or kicker"

Reply:
fixed